distributed-tests/rackspace-server-manager.py

```diff
@@ -39,7 +48,13 @@ def create_node(nova, counts):
         if ip_address is None:
             print 'No IP address assigned!'
             sys.exit(1)
-        print 'The server {0} is waiting at IP address {1}.'.format(count, ip_address)
+        else:
+            ret = ping_node(ip_address)
+            while ret != 0:
```

Comment:
Move this outside the for loop. Put all the IPs into a list. As they come online, pop them out. And only do this for a certain amount of time. 300 seconds perhaps?